checkout_session.go

```diff
@@ -625,6 +637,15 @@ type CheckoutSessionParams struct {
 	//
 	// You can set [`payment_intent_data.setup_future_usage`](https://stripe.com/docs/api/checkout/sessions/create#create_checkout_session-payment_intent_data-setup_future_usage) to have Checkout automatically attach the payment method to the Customer you pass in for future reuse.
 	Customer *string `form:"customer"`
+	// Configure whether a Checkout Session creates a [Customer](https://stripe.com/docs/api/customers) during Session confirmation.
+	//
+	// When a Customer is not created, you can still retrieve email, address, and other customer data entered in Checkout
+	// with [customer_details](https://stripe.com/docs/api/checkout/sessions/object#checkout_session_object-customer_details).
+	//
+	// Sessions that do not create Customers will instead create [Guest Customers](https://support.stripe.com/questions/guest-customer-faq) in the Dashboard.
+	//
+	// Can only be set in `payment` and `setup` mode.
+	CustomerCreation *string `form:"customer_creation"`
```

Comment:
Could you add this to the changelog?

Reply:
They got added in https://github.com/stripe/stripe-go/pull/1403. It seems they accidentally got removed in https://github.com/stripe/stripe-go/pull/1405/files#diff-bcecc69ea00cdcf31411069c0bcb9ef71ab4e490488e20c3690db3244a4c79b5R627 though. Maybe that generation wasn't done against HEAD? Thankfully that didn't get released yet though so not a big deal. I'll add it to the changelog.

Reply:
Oops! Yeah I was doing it against whatever I had in my openapi repo at the time, and I must have not updated that. Sorry 🙈